pr_review_queue.py

```diff
@@ -244,24 +245,65 @@ def create_pr_review_queue(pull_request_list):
     Return a filtered list of pull requests according to these criteria:
         1. CI is green
         2. Not a draft
-        3. No changes requested
-        4. No merge conflicts
+    The resulting pull requests are grouped into four sections:
+        1. ‘Needs reviewer’: ping author about finding a reviewer
+        2. ‘Needs changes’: ping author to do the changes
+        3. ‘Needs review’: ping assigned reviewer/s
+        4. ‘Needs conflict resolution’: ping author to rebase/fix conflicts
     """
-    pr_review_queue = []
-    i = 0
+    needs_reviewer = []
+    needs_changes = []
+    needs_review = []
+    needs_conflict_resolution = []
+
     for pull_request in pull_request_list:
+        # 1. Needs reviewer
         if (pull_request["status"] == "success" and
             pull_request["draft"] == False and
             pull_request["changes_requested"] == False and
+            pull_request["mergeable_state"] != "dirty" and
+            pull_request["requested_reviewers"] == []):
+            entry = (f"*{pull_request['repo']}*:"
+                     f" <{pull_request['html_url']}|{pull_request['title']}>"
+                     f" (+{pull_request['additions']}/-{pull_request['deletions']})"
```

Comment:
I'm not sure if this is really relevant to see - I would rather be interrested e.g. in the age of the PR
e.g.
"(PR age: 4d)"
for example just hours below 24, then just days (and ignore hours to be shorter)

Reply:
The size in lines is, for me at least, a nice indicator of how long I can expect to spend on it, so it's nice to have.  Age is good too though, but maybe more useful is (time since) Last Updated, because if a PR is 2 weeks old but was open as draft and a WIP until a day ago, the age isn't really useful information.

Reply:
Sounds good, the date when a PR was last updated is already part of the values we fetch per PR.
How about a short notation as in:
2d, 3w, 4w+ etc?

Reply:
Made that a follow-up too, so this PR can land.